README.md

```diff
@@ -1,44 +1,49 @@
+- [The archiving web app](http://www.archivers.space/) to track progress through all subsequent stages
 
-**If you are a regular participant**: 
-- Get a role assignment (e.g., Seeder, or Harvester), get account credentials needed for your role, and make sure you have access to the key documents and spreadsheets needed to do the work. The Event/Remote organizers will tell you how proceed to do all this. 
-- Go over the workflow documentation below, in particular the pages corresponding to your role.
+We also rely heavily on the [EDGI Agency & Sub-agency Primers](https://envirodatagov.org/agency-forecasts/) as guides to where to look for important data andweb pages.
+
+We use lots of other docs and tools too; we link to some of them in this document, but you can also explore the [EDGI Github organization](https://github.com/edgi-govdata-archiving/) and especially the [Overview](https://github.com/edgi-govdata-archiving/overview) repository. 
 
-**********************
 ## Plan Overview
-### 1. [Seeders/Sorters](seednsort.md)
-Seeders and Sorters canvass the resources of a given government agency, identifying important URLs. They identify whether those URLs can be crawled by the Internet Archive's webcrawler. If the URLs are crawlable, the Seeders/Sorters nominate them to the End-of-Term (EOT) project, otherwise they add them to the Uncrawlable spreadsheet using the project's Chrome Extension.
 
-### 2. [Researchers](research.md)
-Researchers inspect the "uncrawlable" list to confirm that seeders' assessments were correct (that is, that the URL/dataset is indeed uncrawlable), and investigate how the dataset could be best harvested. [Research.md](research.md) describes this process in more detail. 
+###  [Seeders/Sorters](seednsort.md) 
+Seeders and Sorters canvass the resources of a given government agency, identifying important URLs. They identify whether those URLs can be crawled by the Internet Archive's webcrawler. They use the [Nomination Tool Chrome Extension](https://chrome.google.com/webstore/detail/nominationtool/abjpihafglmijnkkoppbookfkkanklok?hl=en) as well as the [agency primers](https://github.com/edgi-govdata-archiving/agency-primers/blob/master/README.md)  for this work.
 
-*We recommend that a Researchers and Harvesters (see below) work together in pairs, as much communication is needed between the two roles. In some case, one same person will fulfill both roles.*
+### 2. [Researchers](research.md)
+From this stage on, progress is tracked exclusively through the [Data Rescue archivers app](http://archivers.space). Researchers inspect individual URL's/datasets and make a more granular assessment of them, including a preliminary assessment of the type of data, the kinds of tools that might be useful for harvesters, and the importance of the data.  [Research.md](research.md) describes this process in more detail. 
+*Often this step is incorporated into either "Seeding and Sorting" or "Harvesting".*
```

Comment:
Do we want to wrap it into seeding? I'm also not sure of the term "sorting" right now and what exactly that involves